src/main/kotlin/dev/arbjerg/ukulele/command/HelpCommand.kt

```diff
@@ -1,15 +1,35 @@
 package dev.arbjerg.ukulele.command
 
+import dev.arbjerg.ukulele.config.BotProps
 import dev.arbjerg.ukulele.features.HelpContext
 import dev.arbjerg.ukulele.jda.Command
 import dev.arbjerg.ukulele.jda.CommandContext
+import net.dv8tion.jda.api.MessageBuilder
 import org.springframework.stereotype.Component
 
 @Component
 class HelpCommand : Command("help") {
-    override suspend fun CommandContext.invoke() {
-        val command = beans.commandManager[argumentText.trim()] ?: return replyHelp()
-        replyHelp(command)
+    override suspend fun CommandContext.invoke() = when {
+        argumentText.isNotBlank() -> {
+            val command = beans.commandManager[argumentText.trim()]
+            if (command == null) {
```

Comment:
There are three ways you can shorten this block to just 1 or 2 lines

One is to just default to `this`, which is also the default when you provide no arguments

`replyHelp(beans.commandManager[argumentText.trim()] ?: this)`

One is to do an early return with an elvis operator
`beans.commandManager[argumentText.trim()] ?: return replyHelp()`

Another is to shorten the if statement
`if (command == null) replyHelp() else replyHelp(command)`

Reply:
With the first option, I had to use `this.command`; is this the right way, or would something with [qualified this](https://kotlinlang.org/docs/this-expressions.html#qualified-this) be a better solution?

Reply:
I see now that `this` refers to the `CommandContext` receiver (see line 12). `this.command` is correct as it happens to refer to the same instance of `HelpCommand`. Could probably just write `command` though.

Reply:
Good point, changed to `command`.